utils/containers/hash/hash_queue.c.tpl

```diff
@@ -67,3 +67,26 @@ flex_trit_t *hash{SIZE}_queue_at(hash{SIZE}_queue_t *const queue, size_t index)
   }
   return NULL;
 }
+
+retcode_t hash{SIZE}_queue_copy(hash{SIZE}_queue_t const src, hash{SIZE}_queue_t *dest, size_t size) {
```

Comment:
Usually, `copy` functions takes `dest` as first parameter and `src` as second parameter, so it can be a bit confusing